ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -684,7 +684,8 @@ create_attributes_from_template(struct obj_attrs **out, void *template,
 				size_t template_size,
 				struct obj_attrs *parent __unused,
 				enum processing_func function,
-				enum pkcs11_mechanism_id mecha __unused)
+				enum pkcs11_mechanism_id mecha,
```

Comment:
Removal of the __unsed from mecha should be in commit https://github.com/OP-TEE/optee_os/pull/4257/commits/a1ef9669dae5b1ef1ac89f9f5b5ea4fb9c349705 as that starts to use the argument.

Reply:
Done